Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN \
 ADD . ${HOME}
 
 # Add task to $PATH so AWS Batch can run
-COPY task.py /usr/local/bin
+COPY [ "task.py", "version.py", "/usr/local/bin" ]
```

Comment:
This syntax looks weird but is correct. Also woops my commit message has my actual $PATH 😁